src/classes/keyframe_scaler.py

```diff
@@ -52,19 +53,23 @@ def _update_prop(self, prop: dict):
             keyframes = [prop.get("Points", [])]
         for k in keyframes:
             # Scale the X coordinate (frame #) by the stored factor
```

Comment:
Invalid comment. Let's double check all the comments in the function, and update any that are misleading or wrong